hr_expense_invoice/models/hr_expense.py

```diff
@@ -27,6 +27,9 @@ class HrExpense(models.Model):
         comodel_name="account.move",
         inverse_name="source_invoice_expense_id",
     )
+    amount_residual = fields.Monetary(
+        string="Amount Due", compute="_compute_amount_residual", store="True"
```

Comment:
The `store` attribute should be a boolean

Reply:
done